docs/components/board/_index.md

```diff
@@ -42,13 +44,29 @@ If there is no board model for your board:
 
 {{< /alert >}}
 
-{{<resources api="rdk:component:board" type="board">}}
+To use your board component, check whether one of the following models supports it.
+
+For configuration information, click on the model name:
+
+{{< tabs >}}
+{{% tab name="RDK" %}}
+
+{{<resources api="rdk:component:board" type="board" no-intro="true">}}
 
 {{< readfile "/static/include/create-your-own-mr.md" >}}
 
-### Micro-RDK
+{{% /tab %}}
+{{% tab name="micro-RDK" %}}
 
-If you are using the micro-RDK, navigate to [Micro-RDK Board](/build/micro-rdk/board/) for supported model information.
+<!-- prettier-ignore -->
+| Model | Description |
+| ----- | ----------- |
+| [`esp32`](esp32/) | An ESP32 microcontroller |
+
+{{% readfile "/static/include/micro-create-your-own.md" %}}
+
+{{% /tab %}}
+{{< /tabs >}}
 
 ## Attribute configuration
```

Comment:
This section doesn't apply to micro-RDK very well: the fields you can set for elements in the 'analogs' and 'digital_interrupts' arrays are different. The micro-RDK fields are described on the esp32 page: https://docs.viam.com/build/micro-rdk/board/esp32/#analogs (linked to 'analogs'; 'digital_interrupts' is further down).

I was also talking with Susmita this morning about how this 'Attribute configuration' section is a bit odd since normally we describe configs on each model's page, not the main page. In fact, we duplicate this information on model pages - I haven't checked all of them, but it is duplicated on at least 'pi5', 'pi', 'jetson'. There are boards that only support analog (like [numato](https://docs-test.viam.dev/3146/components/board/numato/)) or only digital interrupts (like [UP board](https://docs-test.viam.dev/3146/components/board/upboard/)), in which case their model pages only have the supported section.

I think it could be better to either:
1. (less effort) Just remove this section from the main page. Let each model's page say how to configure analogs and interrupts for that model.
1. (more effort, reduce some duplication) Shorten this section to just describing what analog-to-digital converters and interrupts are, and shorten the sections on the model pages to just describing how to configure analogs and interrupts.

Reply:
yeah I am confused why that's even here to be honest, good catch

Reply:
great!